ts-perf/packages/commands/src/scenario/add.ts

```diff
@@ -47,14 +42,12 @@ const command: Command<AddScenarioOptions> = {
             param: "name",
             description: "",
         },
-        scenarioConfigDirs: {
+        scenarioDir: {
             type: "string",
-            longName: "scenarioConfigDir",
-            alias: "scenarioConfigDirs",
-            defaultValue: () => [],
+            alias: "scenarioConfigDir",
             param: "directory",
-            multiple: true,
-            description: "Paths to directories containing scenario JSON files.",
+            description:
+                "Use <directory> as a location containing individual test scenario folders each with a 'scenario.json'. If not set, uses '~/.tsperf/scenarios', if present.",
```

Comment:
Hopefully we don't end up needing another internal repo again; I made this support multiple so that we could have two places, but it's definitely not needed now that the internal repo is not being used. Definitely cleaner as a single dir, though.

Reply:
Happy to make this a single directory if it's not going to break the existing pipelines.

Reply:
I think you already did, by removing `multiple: true`, right? I did run the pipeline and it was happy with the PR, so this should be fine as-is.

Reply:
I'll do that in a follow-up PR

Reply:
> I think you already did, by removing `multiple: true`, right? I did run the pipeline and it was happy with the PR, so this should be fine as-is.

I only did that for `add`. Every other place that takes `--scenarioDir` is still `multiple: true`.

Reply:
Yeah, I was confused, oops. This definitely makes sense for add, forgot you pointed that out.